### ISSUE TITLE
Add a "Labs" Page

### ISSUE THREAD

**Issue by allella** [hackgvl/hackgreenville-com#138]
The content of that Labs page will be pulled largely from https://hackgvl.github.io/codeforgreenville.github.io/

Since Zach is [working on some Laravel and layout refresh](https://github.com/hackgvl/hackgreenville-com/issues/136), we should keep that work in mind when adding this new page and navigation items.

**Comment by allella** [comment]
The [GitHub Pages for CFG website](https://github.com/hackgvl/codeforgreenville.github.io) has been unpublished and archived.

For reference, here's what it looked like before it was unpublished, but I suppose anybody could clone the repo and open it in their browser to see the same.

![image](https://github.com/hackgvl/hackgreenville-com/assets/1777776/07a15cc5-6b5c-4f83-93d3-023ebbba6559)
![image](https://github.com/hackgvl/hackgreenville-com/assets/1777776/d1a0d110-c11b-4cb3-a5fd-92dcbeaa8db2)

**Comment by oliviasculley** [comment]
I was going to make it similar to [Bogdan's original mockup](https://github.com/hackgvl/codeforgreenville.github.io/issues/17#issuecomment-1267835630) but with the Hackgreenville color palette and style. If you want I can push up what I have so far, progress on it has been a bit slow

**Comment by allella** [comment]
Yeah, put the "Projects" section, or something similar, to screenshots above would be a decent start.

The HTML and CSS for that "Projects" section can be found at  https://github.com/hackgvl/codeforgreenville.github.io

**Comment by allella** [comment]
I should add that we'd probably only highlight the active / green projects in that list, since the rest are archived Code For Greenville projects, or ones we're not actively exploring.

**Comment by allella** [comment]
That active project list mostly lines up with our GitHub repos and the [pinned repos](https://github.com/hackgvl/).

![image](https://github.com/hackgvl/hackgreenville-com/assets/1777776/dea049f1-1640-40b3-a199-032023ab9009)
